wls-gui-wahllokalsystem/src/components/wahlhandlung/TheUnguetilgeWahlscheineVerifyCard.vue

```diff
@@ -209,9 +210,11 @@ async function onSaveAbstimmungsergebnisClicked() {
   isAbstimmungsergebnisFormValid.value = false;
 }
 
-function onWahlscheinnummerChanged(newValue: number) {
-  wahlscheinnummer.value = newValue;
-  resetUngueltigerWahlschein();
+function onWahlscheinnummerChanged(newValue: number | null | undefined) {
+  if (newValue !== undefined) {
+    wahlscheinnummer.value = newValue;
+    resetUngueltigerWahlschein();
+  }
```

Comment:
Auch wenn kein `undefined` aktuell kommen kann, würde ich den Code so anpassen da er zumindest alle Parameter unterstützt.

```suggestion
if (newValue !== undefined) {
    wahlscheinnummer.value = newValue;
  } else {
    wahlscheinnummer.value = null;
  }
  resetUngueltigerWahlschein();
```

Reply:
[a862ea9](https://github.com/it-at-m/Wahllokalsystem/pull/1848/commits/a862ea92b9371c74e08716170bf4bcc3a9ab6739)